First edition glossary (#366)
gvwilson/teachtogether.tech: es/glossary.tex

y otras actividades que requieren que reflexionen y usen nueva información en
tiempo real. Ver también \grefcross{g:passive-learning}{aprendizaje pasivo}.

\gitem{g:inquiry-based-learning}{Aprendizaje basado en la indagación} La práctica de permitir que 
las/los estudiantes hagan sus propias preguntas, establezcan sus propios objetivos y encuentren su propio camino a través de un tema.

\gitem{g:cognitive-apprenticeship}{Aprendizaje cognitivo} Una teoría de

> aprendizaje que enfatiza el proceso de la persona que enseña que transmite habilidades e ideas

Reviewer: "que enseña que transmite" me resulta confuso. a ver así? No sé si el significado es el mismo 

```suggestion
aprendizaje que enfatiza el proceso de enseñanza de quien transmite habilidades e ideas
```

\gitem{g:implementation-science}{Ciencia de implementación} El estudio de cómo traducir
los hallazgos de la investigación a la práctica clínica diaria.

> \gitem{g:cs-unplugged}{Ciencias de la computación desconectada} Un estilo de enseñanza que introduce

Reviewer: ver cómo queda en el capítulo community.tex, quizás así?

```suggestion
\gitem{g:cs-unplugged}{Ciencias de la computación desenchufada} Un estilo de enseñanza que introduce
```

conceptos informáticos utilizando ejemplos y artefactos que no son de programación.

> \gitem{g:synchronous}{Clase sincrónica} tanto la/el docente como las/los estudiantes se conectan 

Reviewer: ```suggestion
\gitem{g:synchronous}{Clase sincrónica} Una clase en la que tanto la/el docente como las/los estudiantes se conectan 
```

a un espacio común virtual al mismo tiempo e interactúan en tiempo real. 

> \gitem{g:asynchronous}{Clase asincrónica} la interacción y comunicación de estudiantes y docentes

Reviewer: ```suggestion
\gitem{g:asynchronous}{Clase asincrónica} Una clase en la que la interacción y comunicación de estudiantes y docentes
```

se hace de forma distribuida, no necesariamente en el mismo momento e intercalada con otras actividades. 

\gitem{g:co-teaching}{Co-enseñanza} Enseñar con otra/otro docente en el
salón de clases.

\gitem{g:externalized-cognition}{Cognición externalizada} El uso de ayuda gráfica,
física o verbal para aumentar el pensamiento.

\gitem{g:cognitivism}{Cognitivismo} Una teoría del aprendizaje que sostiene que los estados 
y procesos mentales pueden y deben incluirse en modelos de aprendizaje. Ver también
\grefcross{g:behaviorism}{conductismo}.

> \gitem{g:community-of-practice}{Comunidad de práctica} Grupo de personas auto organizado y auto mantenido

Reviewer: el sujeto es "grupo" que es singular (aquí agrego una coma, sigo en la sgte línea)

```suggestion
\gitem{g:community-of-practice}{Comunidad de práctica} Grupo de personas auto organizado y auto mantenido,
```

> que comparten y desarrollan un oficio como ser tejedoras/es, músicas/os o programadoras/es. Ver también

Reviewer: ```suggestion
en el que se comparte y desarrolla un oficio como ser tejedoras/es, músicas/os o programadoras/es. Ver también
```

\gitem{g:connectivism}{Conectivismo} Una teoría del aprendizaje que sostiene que el conocimiento se distribuye,
que el aprendizaje es el proceso de navegación, crecimiento y poda de conexiones, y que enfatiza los aspectos

> sociales del aprendizaje que son posibles gracias a Internet.

Reviewer: ```suggestion
sociales del aprendizaje que son posibles gracias a internet.
```

> \gitem{g:pedagogical-content-knowledge}{Conocimiento de la pedagogía del contenido} (\emph{PCK}, por sus siglas en Inglés) 

Reviewer: ```suggestion
\gitem{g:pedagogical-content-knowledge}{Conocimiento de la pedagogía del contenido} (\emph{PCK}, por sus siglas en inglés) 
```

\gitem{g:cs2}{CS2. ciencias de la computación II} Un segundo curso de ciencias de la computación de nivel universitario
que generalmente presenta estructuras de datos básicas como pilas, colas y diccionarios.

> \gitem{g:mooc}{Cursos masivos en línea} (\emph{MOOC}, por sus siglas en Inglés) Un curso en línea diseñado para la inscripción masiva y el estudio asíncrono, que generalmente usa videos grabados y calificaciones automáticas.

Reviewer: ```suggestion
\gitem{g:mooc}{Cursos masivos en línea} (\emph{MOOC}, por sus siglas en inglés) Un curso en línea diseñado para la inscripción masiva y el estudio asíncrono, que generalmente usa videos grabados y calificaciones automáticas.
```

Reviewer: asíncrono o asincrónico? para seguir con el término de arriba

lecciones específicas para audiencias específicas. Ver también
\grefcross{g:educational-psychology}{psicología educacional}.

\gitem{g:plausible-distractor}{Distractor pausible} Una respuesta incorrecta, que parece que podría ser correcta, 
en una \grefcross{g:multiple-choice-questions}{pregunta de opción múltiple}. Ver también
\grefcross{g:diagnostic-power}{poder diagnóstico}.

\gitem{g:split-attention-effect}{Efecto de atención dividida} La disminución que ocurre
en el aprendizaje cuando las/los estudiantes deben dividir su atención entre múltiples 
presentaciones concurrentes de la misma información (por ejemplo, subtítulos y una voz en off).

\gitem{g:dunning-kruger-effect}{Efecto Dunning-Kruger} La tendencia de las personas que solo 

> saben un poco sobre un tema, a estimar incorrectamente su comprensión del mismo.

Reviewer: ```suggestion
saben un poco sobre un tema a estimar incorrectamente su comprensión del mismo.
```

\gitem{g:expertise-reversal}{Efecto inverso de la experiencia} La forma en que
la instrucción que es efectiva para las personas novatas se vuelve ineficaz para
las personas competentes o expertas.

> \gitem{g:faded-example}{Ejemplo desvanecidos} Una serie de ejemplos en los que

Reviewer: es desvanecido o desvanecidos?
y si es "ejemplo" en singular, no sé si es raro "Una serie de ejemplos..."

se borra un número cada vez mayor de pasos clave. Ver
también \grefcross{g:scaffolding}{andamiaje}.

\gitem{g:teaching-to-the-test}{Enseñar para el examen} Cualquier método de ``educación''
que se centre en preparar a las/los estudiantes para aprobar los exámenes estandarizados, en lugar de aprender realmente.

\gitem{g:active-teaching}{Enseñanza activa} Un enfoque de la instrucción en el que la/el docente

> actúa sobre la nueva información adquirida de las/los estudiantes mientras enseña (ej.\ cambiando

Reviewer: ```suggestion
actúa sobre la nueva información adquirida de las/los estudiantes mientras enseña (p. ej.\ cambiando
```

dinámicamente un ejemplo o reorganizando el orden previsto del contenido).
Ver también \grefcross{g:passive-teaching}{enseñanza pasiva}.

\gitem{g:direct-teaching}{Enseñanza directa} Ver \grefcross{g:direct-instruction}{instrucción directa}.

\gitem{g:passive-teaching}{Enseñanza pasiva} Un enfoque a la enseñanza en el que la/el docente no ajusta el ritmo o los ejemplos, 

> o no actúa de acuerdo con los comentarios de las/los estudiantes, durante la lección.  

Reviewer: ```suggestion
o no actúa de acuerdo con los comentarios de las/los estudiantes durante la lección.  
```

Ver también \grefcross{g:active-teaching}{enseñanza activa}.

\gitem{g:free-range-learner}{Estudiante free-range} Alguien que aprende fuera de un aula institucional 
con un plan de estudios y tareas obligatorias. 
(Quienes usan este término, ocasionalmente se refieren a las/los estudiantes en las aulas 

> como ``estudiantes battery-farmed'', pero nosotros no lo hacemos porque sería grosero).

Reviewer: No sé si se entiende lo de battery-farmed, sin el contexto de la explicación de Greg.

\gitem{g:subgoal-labeling}{Etiquetado de sub\-objetivos} Dar nombres a cada paso en una descripción paso a paso 
de un proceso de resolución de problemas.

\gitem{g:formative-assessment}{Evaluación formativa} Evaluación que se lleva a cabo
durante una clase para dar retroalimentación tanto a estudiantes como a docentes
sobre la comprensión real. Ver también \grefcross{g:summative-assessment}{evaluación sumativa}.

\gitem{g:summative-assessment}{Evaluación sumativa} Evaluación que se realiza al final de una 
lección para determinar si se ha realizado el aprendizaje deseado.

\gitem{g:expert}{Experta/o} Alguien que puede diagnosticar y manejar situaciones inusuales, 
sabe cuándo no se aplican las reglas habituales y tiende a reconocer soluciones en lugar de razonarlas.
Ver también \grefcross{g:competent-practitioner}{practicante competente}
y \grefcross{g:novice}{novata/o}.

> \gitem{g:productive-failure}{Falla productiva} Una situación en la que a las/los estudiantes se les dan deliberadamente, 

Reviewer: ```suggestion
\gitem{g:productive-failure}{Falla productiva} Una situación en la que a las/los estudiantes se les dan deliberadamente 
```

> problemas que no se pueden resolver con el conocimiento que tienen y deben salir y adquirir nueva información para progresar.

Reviewer: ```suggestion
problemas que no se pueden resolver con el conocimiento que tienen, por lo que deben buscar y adquirir nueva información para progresar.
```

Ver también \grefcross{g:zpd}{Zona de desarrollo próximo}.

> \gitem{g:false-beginner}{Falsa/o principiante} Las/los falsas/os principiantes comienzan en el mismo punto que los principiantes 

Reviewer: ```suggestion
\gitem{g:false-beginner}{Falsa/o principiante} Aquellas personas que comienzan en el mismo punto que las/los principiantes 
```

> verdaderos (es decir, en una evaluación inicial mostrarán el mismo nivel de competencia) pero pueden avanzar mucho más rápido.

Reviewer: ```suggestion
verdaderas/os (es decir, en una evaluación inicial mostrarán el mismo nivel de competencia) pero pueden avanzar mucho más rápido.
```

Por ejemplo, alguien que ha estudiado un idioma antes pero lo está aprendiendo nuevamente. 

\gitem{g:hypercorrection}{Fenómeno de hipercorrección} Cuanto más cree alguien
que su respuesta en un examen era correcta, más probabilidades hay de que no

> repita el error una vez que descubre que, de hecho, estaba equivocada/do.

Reviewer: ```suggestion
repita el error una vez que descubre que, de hecho, se había equivocado.
```

\gitem{g:flow}{Flujo} La sensación de estar completamente inmerso en una actividad,
frecuentemente asociada con una alta productividad.

\gitem{g:chunking}{Fragmentación} El acto de agrupar conceptos relacionados
que pueden almacenarse y procesarse como una sola unidad.

\gitem{g:fuzz-testing}{Fuzz testing} Una técnica de prueba de software
basada en generar y enviar datos aleatorios.

\gitem{g:hashing}{Hashing} Generar una clave digital pseudoaleatoria condensada a partir de datos; cualquier entrada específica produce la misma salida, pero es muy probable que diferentes entradas produzcan diferentes salidas.

\gitem{g:inclusivity}{Inclusión} Trabajar activamente para incluir
personas con diversos antecedentes y necesidades.

> \gitem{g:learned-helplessness}{Indefensión aprendida} Una situación en la que, las personas que son sometidas repetidamente a 

Reviewer: ```suggestion
\gitem{g:learned-helplessness}{Indefensión aprendida} Una situación en la que las personas que son sometidas repetidamente a 
```

Reviewer: No sé si se lee mejor así:

Una situación que ocurre cuando las personas no intentan defenderse de comentarios negativos, a pesar de poder hacerlo, debido a que en ocasiones anteriores han sido repetidamente sometidas a comentarios negativos de los que no podían escaparse.

> comentarios negativos de los cuales no tienen forma de escapar, aprenden a ni siquiera intentar escapar cuando pueden.

Reviewer: esta coma sobraría pero a su vez es muy larga la oración

```suggestion
comentarios negativos de los cuales no tienen forma de escapar aprenden a ni siquiera intentar escapar cuando pueden.
```

\gitem{g:direct-instruction}{Instrucción directa} Un método de enseñanza centrado
en un diseño curricular meticuloso dictado a través de guiones pre-escritos.

\gitem{g:peer-instruction}{Instrucción por pares} Un método de enseñanza en el que la/el docente hace una pregunta y 

> luego las/los estudiantes se comprometen con una primera respuesta, discuten las respuestas con sus compañeras/os y 

Reviewer: "commit" - Se repite "comprometen" que es una palabra llamativa.
fijate, no estoy 100% segura

```suggestion
luego las/los estudiantes arriesgan una primera respuesta, discuten las respuestas con sus compañeras/os y 
```

> se comprometen con una respuesta revisada.

Reviewer: ```suggestion
finalmente se afianzan en una respuesta revisada.
```

crear variantes de diseños tradicionales usando herramientas de dibujo por computadora.

\gitem{g:intuition}{Intuición} La capacidad de comprender algo de inmediato,
sin necesidad aparente de razonamiento consciente.

\gitem{g:concept-inventory}{Inventario de conceptos} Una prueba diseñada para determinar
qué tan bien una/un estudiante comprende un dominio. A diferencia de la mayoría de las pruebas realizadas por docentes,
los inventarios de conceptos se basan en una extensa investigación y validación.

\gitem{g:jugyokenkyu}{Jugyokenkyu} Literalmente ``estudio de lección'', un conjunto de prácticas que incluye hacer que 
las/los docentes se observen rutinariamente entre sí y discutan las lecciones que imparten para compartir conocimientos y 
mejorar habilidades.

\gitem{g:demonstration-lesson}{Lección de demostración} Una lección dictada por una/un docente a estudiantes reales
mientras otras/os docentes observan para aprender nuevas técnicas de enseñanza.

\gitem{g:read-cover-retrieve}{Leer-cubrir-recordar} Una práctica de estudio
en la que la/el estudiante cubre hechos o términos clave durante un primer paso por el material y 
luego verifica cuánto recuerda en un segundo paso.

> \gitem{g:commons}{Los Comunes} algo gestionado conjuntamente por una comunidad

Reviewer: ver cómo queda en community.tex

\gitem{g:metacognition}{Metacognición} Pensar acerca de pensar.

> \gitem{g:mooc-sigla}{MOOC} Ver \grefcross{g:mooc}{cursos masivos en linea}.

Reviewer: ```suggestion
\gitem{g:mooc-sigla}{MOOC} Ver \grefcross{g:mooc}{cursos masivos en línea}.
```

\gitem{g:deficit-model}{Modelo deficitario} La idea de que algunos grupos

> están subrepresentados en informática (o algún otro campo) porque sus integrantes

Reviewer: ```suggestion
están subrepresentados en informática (o en algún otro campo) porque sus integrantes
```

> \gitem{g:seo}{Motor de optimización de posicionamiento en buscadores} (\emph{SEO}, por sus siglas en Inglés)  Aumentar la cantidad y la calidad del tráfico del sitio web al hacer que las páginas sean más fáciles de encontrar o parezcan más importantes para los motores de búsqueda.

Reviewer: ```suggestion
\gitem{g:seo}{Motor de optimización de posicionamiento en buscadores} (\emph{SEO}, por sus siglas en inglés)  Aumentar la cantidad y la calidad del tráfico del sitio web al hacer que las páginas sean más fáciles de encontrar o parezcan más importantes para los motores de búsqueda.
```

> \gitem{g:guided-notes}{Notas guiadas} Notas preparadas por la/el docente, para sus estudiantes, que donde se destaca la 

Reviewer: ```suggestion
\gitem{g:guided-notes}{Notas guiadas} Notas preparadas por la/el docente, para sus estudiantes, donde se destaca la 
```

> información clase de una clase o lección. 

Reviewer: ```suggestion
información clave de una clase o lección. 
```

\gitem{g:novice}{Novata/o} Alguien que aún no ha construido un modelo mental utilizable de un dominio. 

> Ver también \grefcross{g:competent-practitioner}{Practicantes competentes} y \grefcross{g:expert}{experta/o}.

Reviewer: ```suggestion
Ver también \grefcross{g:competent-practitioner}{practicantes competentes} y \grefcross{g:expert}{experta/o}.
```

\gitem{g:learning-objective}{Objetivo de aprendizaje} Qué está intentando lograr enseñar la lección.

\gitem{g:reusability-paradox}{Paradoja de la reusabilidad} Sostiene que cuanto más reutilizable es una lección, 
es menos efectiva pedagógicamente.

\gitem{g:learner-persona}{Persona tipo} Una breve descripción de una/un estudiante objetivo tipo para una 
lección que incluye: sus antecedentes generales, lo que ya sabe, lo que quiere hacer, cómo la lección le ayudará 
y cualquier necesidad especial que puedan tener.

\gitem{g:legitimate-peripheral-participation}{Participación inicial legítima} La participación de las personas recién 
llegadas al grupo en tareas simples y de bajo riesgo que una  
\grefcross{g:community-of-practice}{comunidad de práctica} reconoce como contribuciones válidas.

\gitem{g:computational-thinking}{Pensamiento computacional} Pensar la
resolución de problemas en formas inspiradas en la programación (aunque el término se usa de muchas otras maneras).

\gitem{g:think-pair-share}{Piensa-trabaja en pareja-comparte} Un método de colaboración
en el que cada persona piensa individualmente sobre una pregunta o problema,

> luego se junta con una/un compañera/o para compartir ideas, y luego una persona de

Reviewer: ```suggestion
luego se junta con un par a compartir ideas, y luego una persona de
```

\gitem{g:competent-practitioner}{Practicante competente} Alguien que puede
realizar tareas normales con un esfuerzo normal en circunstancias normales. Ver también

> \grefcross{g:novice}{principiante} y \grefcross{g:expert}{experta/o}.

Reviewer: está bien que novice es  principiante?

Reviewer: pondría novata/o porque parece ser la entrada que tiene la definición

\gitem{g:multiple-choice-questions}{Pregunta de opción múltiple} Tipo de pregunta cerrada simple que permite 
a las/los estudiantes seleccionar una respuesta de una lista predefinida de opciones.

\gitem{g:objects-first}{Primero objetos} Un enfoque para enseñar programación donde
objetos y clases se introducen temprano.

\gitem{g:beginner}{Principiante}. Ver \grefcross{g:novice}{novata/o}

\gitem{g:absolute-beginner}{Principiante absoluto} Alguien que nunca se ha
encontrado con los conceptos o material antes. El término se usa en distinción para
\grefcross{g:false-beginner}{falsa/o principiante}.

\gitem{g:preparatory-privilege}{Privilegio de preparación} La ventaja de provenir de un entorno que proporciona más 
preparación para una tarea de aprendizaje en particular que otras.

> \gitem{g:parsons-problem}{Problemas de Parson} Una técnica de evaluación desarrollada

Reviewer: ```suggestion
\gitem{g:parsons-problem}{Problemas de Parsons} Una técnica de evaluación desarrollada
```

por Dale Parsons y otros en la que las/los estudiantes reorganizan el material dado
para construir una respuesta correcta a una pregunta.~\cite{Pars2006}.

\gitem{g:pair-programming}{Programación en pareja} Una práctica de desarrollo de software en la que 
dos programadoras/es comparten una computadora. Una persona (la/el piloto) escribe, mientras que la otra 
(la/el navegante) ofrece comentarios y sugerencias en tiempo real. 
La programación en pareja a menudo se usa como práctica docente en las clases de programación.

\gitem{g:live-coding}{Programación en vivo} El acto de enseñar programación
escribiendo software frente a las/los estudiantes a medida que avanza la lección.

\gitem{g:conversational-programmer}{Programadora/or conversacional} Alguien que necesita saber lo suficiente sobre 

> computación para tener una conversación significativa con una/un programadora/or, pero que no va a programar por sí misma/o.

Reviewer: ```suggestion
computación para tener una conversación significativa con una/un programadora/or, pero que no va a programar por su cuenta.
```

\gitem{g:learning-outcome}{Resultado de aprendizaje} Qué es lo que la lección realmente logra enseñar.

\gitem{g:calibrated-peer-review}{Revisión por pares calibrada} Hacer que estudiantes comparen sus
revisiones del trabajo de ejemplo con las de una/un docente antes de que
se les permita revisar el trabajo de sus pares.

> \gitem{g:impostor-syndrome}{Síndrome de la/el impostora/or} Un sentimiento de inseguridad sobre

Reviewer: queda rarísimo! lo de "de la" me parece que debería ser así por el acuerdo. pero la otra parte en verdad no sé, es que queda rarito

```suggestion
\gitem{g:impostor-syndrome}{Síndrome de la impostora o impostor} Un sentimiento de inseguridad sobre
```

los logros propios, que se manifiesta como un miedo a ser expuesta/o como un fraude.

> \gitem{g:lms}{Sistema de gestión de aprendizaje} (\emph{LMS}, por sus siglas en Inglés): Una aplicación para registrar 

Reviewer: ```suggestion
\gitem{g:lms}{Sistema de gestión de aprendizaje} (\emph{LMS}, por sus siglas en inglés): Una aplicación para registrar 
```

\gitem{g:tutorial}{Tutorial} Una lección destinada a ayudar a alguien a
mejorar su comprensión general de un tema.

> \gitem{g:twitch-coding}{Sacudir la programación}Hacer que un grupo de personas decida momento a momento

Reviewer: ```suggestion
\gitem{g:twitch-coding}{Sacudir la programación} Hacer que un grupo de personas decida momento a momento
```

o línea por línea qué agregarle a un programa.

\gitem{g:end-user-teacher}{Usuaria/o final docente} Por analogía con

> \grefcross{g:end-user-programmer}{usuaria/o final programadora/or},

Reviewer: sí, feo... no sé. Me pasa igual o peor con impostora/or !!

alguien que enseña con frecuencia, pero cuya ocupación principal no es la enseñanza,
que tiene poca o ninguna experiencia en pedagogía y que puede trabajar fuera de las aulas institucionales.

\gitem{g:end-user-programmer}{Usuaria/o final programadora/or} Alguien que no se considera
una/un programadora/or, pero que, sin embargo, escribe y depura software, como por ejemplo, 
un artista que crea macros complejas para una herramienta de dibujo.

> \gitem{g:zpd}{Zona de desarrollo próximo} (\emph{ZPD}, por sus siglas en Inglés) Incluye el conjunto de problemas 

Reviewer: ```suggestion
\gitem{g:zpd}{Zona de desarrollo próximo} (\emph{ZPD}, por sus siglas en inglés) Incluye el conjunto de problemas 
```
